specs/002-regression-baselines/baseline-format.md

```diff
@@ -0,0 +1,192 @@
+# Baseline JSON file format specification
+
+**Feature**: Static series regression baselines  
+**Version**: 1.0  
+**Date**: October 6, 2025
+
+## Overview
+
+This document defines the JSON schema for regression baseline files. These files store canonical indicator outputs for comparison against current test results to detect unintended behavioral drift.
+
+## File naming convention
+
+- Pattern: `{IndicatorName}.{ScenarioName}.json`
+- Example: `Sma.Standard.json`, `Macd.Standard.json`
+- Location: `tests/indicators/.baselines/`
+- Case sensitivity: Matches indicator class name (e.g., `Sma`, not `SMA`)
+
+## JSON schema
```

Comment:
This can instead be a more direct mapping of the result class for direct deserialization